delivery_service.rb

```diff
@@ -1,2 +1,37 @@
+require_relative 'car'
+require_relative 'bike'
+require_relative 'vehicle_availabilities'
+
+
 class DeliveryService
-end
\ No newline at end of file
+  extend VehicleAvailabilities
+
+  def initialize
```

Comment:
you can use something like that to create vehicle park array 
def initialize
    @bikes = Array.new('array size here')) { Bike.new }
    @cars = Array.new('array size here')) { Car.new }
  end

Reply:
done